content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md

```diff
@@ -0,0 +1,130 @@
+    $ helm template \
+        --set global.proxy.tracer="lightstep" \
+        --set global.tracer.lightstep.address="<Satellite Address>" \
+        --set global.tracer.lightstep.accessToken="<LightStep Access Token>" \
+        --set global.tracer.lightstep.secure=true \
+        --set global.tracer.lightstep.cacertPath="/etc/lightstep/cacert.pem" \
+        install/kubernetes/helm/istio \
+        --name istio --namespace istio-system > $HOME/istio.yaml
+    $ kubectl create namespace istio-system
+    $ kubectl apply -f $HOME/istio.yaml
+    {{< /text >}}
+
+1.  Create a secret in the default namespace (or whichever namespace you are deploying the Bookinfo application into)
+    with the Certificate Authority certificate used for verifying TLS communication with the Satellite pool.
+
+    {{< text bash >}}
+    $ CACERT=$(cat Cert_Auth.crt | base64) # Cert_Auth.crt contains the necessary CACert
+    $ NAMESPACE=default
+    {{< /text >}}
+
+    ```bash
+    $ cat <<EOF | kubectl apply -f -
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: lightstep.cacert
+        namespace: $NAMESPACE
+        labels:
+          app: lightstep
+      type: Opaque
+      data:
+        cacert.pem: $CACERT
+    EOF
+    ```
+
+1.   Deploy the [Bookinfo](/docs/examples/bookinfo/) sample application.
+
+## Visualize trace data
+
+1.  Send traffic to the sample application.
+
+    For the
+    [Bookinfo](/docs/examples/bookinfo/#determining-the-ingress-ip-and-port)
+    sample, visit `http://$GATEWAY_URL/productpage` in your web browser
+    or issue the following command:
+
+    {{< text bash >}}
+    $ curl http://$GATEWAY_URL/productpage
+    {{< /text >}}
+
+1.  Load the LightStep [𝑥]PM web UI then navigate to Explorer and find the query bar at the top. The query bar allows
+    you to interactively filter results by a Service, Operation, and/or Tag values. Select `productpage.default` from
+    the Service drop-down list and click "Run". You should see something similar to the following:
+
+{{< image width="100%" ratio="50%"
+    link="./istio-tracing-list-lightstep.png"
+    caption="Explorer"
+    >}}
+
+If you click on the first row in the table of example traces below the latency histogram, you should see the details
+corresponding to your refresh of the `/productpage`. The page should look something like this:
+
+{{< image width="100%" ratio="50%"
+    link="./istio-tracing-details-lightstep.png"
+    caption="Detailed Trace View"
+    >}}
+
+As you can see, the trace is comprised of a set of spans,
+where each span corresponds to a Bookinfo service invoked during the execution of a `/productpage` request.
+
+Every RPC is represented by two spans in the trace. For example, the call from `productpage` to `reviews` starts
+with the span labeled with operation `reviews.default.svc.cluster.local:9080/*` and service
+`productpage.default: proxy client`, which represents the client-side span for the call. It took 15.30 ms. The second
```

Comment:
Will readers obtain the exact same response times? If you are referencing the values on the screenshots, then make the reference explicit to avoid confusion. The fix continues:
```suggestion
`productpage.default: proxy client` service, which represents the client-side span of the call. The screenshot shows that the call took 15.30 ms. The second
```